install.md

```diff
@@ -167,10 +167,16 @@ profile: full
 ceip_policy_disclosed: FALSE-OR-TRUE-VALUE # Installation fails if this is not set to true. Not a string.
 buildservice:
   kp_default_repository: "KP-DEFAULT-REPO"
-  kp_default_repository_username: "KP-DEFAULT-REPO-USERNAME"
-  kp_default_repository_password: "KP-DEFAULT-REPO-PASSWORD"
-  tanzunet_username: "TANZUNET-USERNAME"
-  tanzunet_password: "TANZUNET-PASSWORD"
+  kp_default_repository_username: "KP-DEFAULT-REPO-USERNAME"    # Mutually exclusive with kp_default_repository_secret
+  kp_default_repository_password: "KP-DEFAULT-REPO-PASSWORD"    # Mutually exclusive with kp_default_repository_secret
+  kp_default_repository_secret:                                 # Mutually exclusive with kp_default_repository_username/kp_default_repository_password
+    name: "KP-DEFAULT-REPO-SECRET-NAME"
+    namespace: "KP-DEFAULT-REPO-SECRET-NAMESPACE"
+  tanzunet_username: "TANZUNET-USERNAME"                        # Mutually exclusive with tanzunet_secret
+  tanzunet_password: "TANZUNET-PASSWORD"                        # Mutually exclusive with tanzunet_secret
+  tanzunet_secret:                                              # Mutually exclusive with tanzunet_username/tanzunet_password
```

Comment:
I wonder if we should simplify the values by only providing inline secrets or inline secret refs to make copy and pasting as productive as possible.

Reply:
Probably only show inline secrets because the steps above do not instruct for the kp-repo-secret to be created.

Reply:
@matthewmcnew How about this approach?